src/Microsoft.Health.Fhir.SqlServer.Api/Controllers/SchemaController.cs

```diff
@@ -78,11 +83,14 @@ public FileContentResult SqlScript(int id)
         [HttpGet]
         [AllowAnonymous]
         [Route(KnownRoutes.Compatibility)]
-        public ActionResult Compatibility()
+        public async Task<IActionResult> Compatibility()
         {
             _logger.LogInformation("Attempting to get compatibility");
 
-            throw new NotImplementedException(Resources.CompatibilityNotImplemented);
+            int minVersion = (int)_schemaInformation.MinimumSupportedVersion;
```

Comment:
This is going to vary based on the instance that you happen to hit if the code packages are different.

Reply:
Updated in the last commit to fetch minVersion too from InstanceSchema table